parsera/page.py

```diff
@@ -21,11 +22,12 @@ class ProxySettings(TypedDict, total=False):
 class PageLoader:
     def __init__(
         self,
-        browser: Literal["firefox", "chromium"] = "firefox",
+        browser_name: Literal["firefox", "chromium"] = "firefox",
+        browser: Browser | None = None,
```

Comment:
Let's keep the old interface, make the browser `string` or `Browser` type, and behave conditionally on input.